test/drbgtest.c

```diff
@@ -764,8 +767,9 @@ static int test_rand_drbg_reseed(void)
     /*
      * Test whether all three DRBGs are reseeded by RAND_add()
      */
+    t = time(NULL);
```

Comment:
Here you should really add a comment which explains that the `before_reseed` time has to be measured here and passed into the `test_drbg_reseed()` test, because the `master` DRBG gets already reseeded in `RAND_add()`, whence the check for the condition
`before_reseed <= master->reseed_time`  will fail if the time value happens to increase between the `RAND_add()` and the `test_drbg_reseed()` call.
(Might be unlikely in general, but not so unlikely during a debug session).

Reply:
I stole your words...

Reply:
Oh, you did not steel it, it was a gift ;-)